### PR TITLE
Replaced hardcoded strings with resource strings - merge to master

### PR DESCRIPTION
**Pull request by MariskaIJpelaar**
Converted (hopefully) all hardcoded strings to resource strings.
Fixed small resource error (mipmap files)
Requires review before merging with master

TODO:
  - Spelling and consistency check
  - Add Dutch as a language
  - make 'settings' so the language can be changed

### CODE REVIEW

File: app/src/main/java/com/sebastiaan/xenopelthis/ui/main/SectionsPagerAdapter.java [hunk 1]
```diff
@@ -1,21 +1,31 @@
 package com.sebastiaan.xenopelthis.ui.main;
```
Comment (MariskaIJpelaar) [inline app/src/main/java/com/sebastiaan/xenopelthis/ui/main/SectionsPagerAdapter.java:1]: Could you review this file please?

File: app/src/main/java/com/sebastiaan/xenopelthis/ui/main/activity/main/MainActivity.java [hunk 2]
```diff
@@ -32,7 +32,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
         model = ViewModelProviders.of(this).get(MainViewModel.class);
 
-        SectionsPagerAdapter sectionsPagerAdapter = new SectionsPagerAdapter(getSupportFragmentManager());
+        SectionsPagerAdapter sectionsPagerAdapter = new SectionsPagerAdapter(getSupportFragmentManager(), this);
```
Comment (MariskaIJpelaar) [inline app/src/main/java/com/sebastiaan/xenopelthis/ui/main/activity/main/MainActivity.java:35]: Could you review this file please?

File: app/src/main/java/com/sebastiaan/xenopelthis/ui/product/view/dialog/OverrideDialog.java [hunk 3]
```diff
@@ -1,6 +1,7 @@
 package com.sebastiaan.xenopelthis.ui.product.view.dialog;
```
Comment (MariskaIJpelaar) [inline app/src/main/java/com/sebastiaan/xenopelthis/ui/product/view/dialog/OverrideDialog.java:1]: Could you review this file, please?

File: app/src/main/java/com/sebastiaan/xenopelthis/ui/supplier/view/dialog/OverrideDialog.java [hunk 4]
```diff
@@ -1,6 +1,7 @@
 package com.sebastiaan.xenopelthis.ui.supplier.view.dialog;
```
Comment (MariskaIJpelaar) [inline app/src/main/java/com/sebastiaan/xenopelthis/ui/supplier/view/dialog/OverrideDialog.java:1]: Could you review this file please?

**Comment by MariskaIJpelaar** [comment]
Added some comments on the files I want to have reviewed specifically. @Sebastiaan-Alvarez-Rodriguez

**Review by Sebastiaan-Alvarez-Rodriguez — APPROVED** [review]
Your current way of fetching resource strings is fine by me, as long as it works well.
The way I know of is `getResources().getString(R.string.mystring);`. 
Maybe worth investigating.

**Pull request merged**